boards/samr21-xpro/include/periph_conf.h

```diff
@@ -160,15 +159,15 @@ static const pwm_conf_t pwm_config[] = {
 #if PWM_0_EN
     {TCC1, {
         /* port , pin, AF, chan */
-        {(PortGroup *)0x41004400, 6, 4, 0},
-        {(PortGroup *)0x41004400, 7, 4, 1}
+        {GPIO_PIN(PA,6), GPIO_MUX_E, 0},
```

Comment:
Space after comma.

And, as usual, I'd like to suggest the use symbols instead of literals. Not because it's in the coding conventions, but for the reasons why it is there. ;-)